src/components/RiddleCard.tsx

```diff
@@ -67,7 +59,7 @@ export const RiddleCard: FC<Props> = ({
 			>
 				<CardMedia
 					component="img"
-					image={image}
+					image={image === undefined ? '../../public/RiddlePreview.svg' : image}
```

Comment:
Zde se dá pužít nullish coalescing a absolutní cesta.
```suggestion
image={image ?? 'public/RiddlePreview.svg'}
```